war/src/main/java/io/lumeer/engine/rest/CollectionService.java

```diff
@@ -121,70 +116,103 @@ public void init() {
     */
    @GET
    @Path("/")
-   public List<String> getAllCollections() {
-      List<String> collections = new ArrayList<>();
+   public List<Collection> getCollections(@QueryParam("limit") int limit) {
```

Comment:
I think we need proper paging here, not only `limit`. I would suggest using `page` and `size` parameters.